src/forms/wods_form.cr

```diff
@@ -0,0 +1,8 @@
+class WodForm < Wod::BaseForm
+  allow description
+
+  def prepare
+    super
+    date.value = Time.now.at_beginning_of_day
```

Comment:
Nice. That's a clever way of doing dates. I may make a `Date` abstraction that does exactly this at some point but I'm not sure